README.md

```diff
@@ -1 +1,206 @@
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://kaghenimbale.github.io/Capstone-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install Git on your computer following the steps [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- Install node js following the steps [here](https://www.pluralsight.com/guides/getting-started-with-nodejs)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Kaghenimbale/Capstone-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  Open the index.html in the browser
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```  cd my-project
+  npx hint .
+  npx stylelint "**/*.{css,scss}"
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+    github page and choose the name of the branch to deploy
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Kaghenimbale)
+- Twitter: [@twitterhandle](https://twitter.com/MbaleKagheni)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kagheni-mbale-401b90240/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Complete all the navigation bar project]**
+- [ ] **[Adjust the styling]**
+- [ ] **[Add others functionality using Javascript]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a start ⭐️ If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thank you Microverse and my friend who always explain me for preparing this project! clapclapclap
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Kaghenimbale/my-portfolio/commit/55ba2a9f34c66b7803d1963fff7e3a1cdafbd12c)) licensed.
```

Comment:
- [ ] You are not supposed to add your commit link here :fearful: You are supposed to add your LICENSE file link :+1: After adding it this section will look like this :+1:
```javascript
This project is [MIT](./LICENSE) licensed.
```
- [ ] After making the above change your link won't work because you don't have a LICENSE file in your repository :thinking: So you can add a LICENSE file in your repository by following these given [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1: